data/core/units/humans/Peasant.cfg

```diff
@@ -15,7 +15,7 @@
     advances_to=Spearman,Bowman
     cost=8
     usage=fighter
-    description= _ "Peasants are the backbone of the rural economy, and the soldiers of last resort. While not warlike by nature, they will stubbornly defend their homes. However, if you’re hurling peasants at your foes, you’re clearly out of superior forces."
+    description= _ "Peasants are the backbone of the rural economy, and the soldiers of last resort. While not warlike by nature, they will stubbornly defend their homes. Their lack of training and discipline makes them unfit to hold a line, though; their reliance on farming tools as weaponry means they are only on the field of battle under the most desperate situations."
```

Comment:
.. unfit to hold a line, **and** their reliance ..

or

.. unfit to hold a line**. Their** reliance ..